pkg/scheduler/cache/cache.go

```diff
@@ -538,20 +541,7 @@ func newSchedulerCache(config *rest.Config, schedulerName string, defaultQueue s
 
 // Run  starts the schedulerCache
 func (sc *SchedulerCache) Run(stopCh <-chan struct{}) {
-	go sc.podInformer.Informer().Run(stopCh)
-	go sc.nodeInformer.Informer().Run(stopCh)
-	go sc.podGroupInformerV1beta1.Informer().Run(stopCh)
-	go sc.pvInformer.Informer().Run(stopCh)
-	go sc.pvcInformer.Informer().Run(stopCh)
-	go sc.scInformer.Informer().Run(stopCh)
-	go sc.queueInformerV1beta1.Informer().Run(stopCh)
-	go sc.quotaInformer.Informer().Run(stopCh)
-	go sc.cpuInformer.Informer().Run(stopCh)
-
-	if options.ServerOpts.EnablePriorityClass {
-		go sc.pcInformer.Informer().Run(stopCh)
-	}
-
+	sc.informerFactory.Start(stopCh)
```

Comment:
have you tested it? l think `informerFactory` doesn't contain volcano related informers?

Reply:
En, I forgot to add volcano informerFactory.
I have added it.